plugins/building/gradle.properties

```diff
@@ -0,0 +1,4 @@
+POM_ARTIFACT_ID=mapbox-android-building-plugin
```

Comment:
To keep consistency around plugins, the artifact id should be `mapbox-android-plugin-building`